src/main/kotlin/org/treeWare/proto3/message/Proto3CaseConverters.kt

```diff
@@ -0,0 +1,11 @@
+package org.treeWare.proto3.message
```

Comment:
Move this to the -core repo, along with the `snakeCaseToKebabCase()` function from the -server repo. When moved, the `Proto3` prefix should be dropped from the current file name. Also add `Case` to the first 2 functions (similar to the existing `snakeCaseToKebabCase()` function).